src/patternfly/components/Slider/slider.scss

```diff
@@ -19,12 +19,12 @@
 
   // step tick
   --#{$slider}__step-tick--InsetBlockStart: var(--pf-t--global--spacer--md);
-  --#{$slider}__step-tick--Width: #{pf-size-prem(4px)};
+  --#{$slider}__step-tick--Width: 0.15rem;
   --#{$slider}__step-tick--Height: #{pf-size-prem(4px)};
-  --#{$slider}__step-tick--BackgroundColor: var(--pf-t--global--icon--color--subtle);
+  --#{$slider}__step-tick--BackgroundColor: var(--pf-t--global--icon--color--on-disabled);
```

Comment:
now that i'm thinking about this more, we should probably use `--pf-t--global--icon--color--nonstatus--on-gray--default` here. It will still look darker than the design screenshot in the issue (I didn't update the tick color token in the screenshot!).

Reply:
Updated 🚀 https://patternfly-pr-6981.surge.sh/components/slider